packages/block-library/src/rss/index.php

```diff
@@ -31,6 +31,28 @@ function render_block_core_rss( $attributes ) {
 
 	$rss_items  = $rss->get_items( 0, $attributes['itemsToShow'] );
 	$list_items = '';
+
+	$link_attributes = '';
+	$rel_attributes  = array();
+
+	if ( ! empty( $attributes['openInNewTab'] ) ) {
+		$link_attributes .= ' target="_blank"';
+	}
+
+
+	if ( ! empty( $attributes['linkRel'] ) ) {
+		$rel_attributes = explode( ' ', $attributes['linkRel'] );
+		$rel_attributes = array_filter( array_map( 'sanitize_html_class', $rel_attributes ) );
+
+		if ( ! empty( $rel_attributes ) ) {
+			$link_attributes .= ' rel="' . esc_attr( implode( ' ', $rel_attributes ) ) . '"';
+		}
```

Comment:
Do we really need all this sanitization for escaped value? Could this be something like:

```php
$link_attributes .= ' rel="' . esc_attr( trim( $attributes['rel'] ) ) . '"';
```